src/StreamJsonRpc/Reflection/MessageFormatterRpcMarshaledContextTracker.cs

```diff
@@ -87,6 +87,44 @@ private enum MarshalMode
             MarshallingRealObject = 1,
         }
 
+        internal static (JsonRpcProxyOptions ProxyOptions, JsonRpcTargetOptions TargetOptions)? GetMarshalOptionsForType(Type type)
+        {
+            Requires.NotNull(type, nameof(type));
```

Comment:
No need to test for null inputs for internal methods since we're using nullable-type reference annotations which would produce a build warning (which would break PR/CI builds).